rpc/rpcserver/server.go

```diff
@@ -2322,7 +2371,9 @@ func (s *loaderServer) SubscribeToBlockNotifications(ctx context.Context, req *p
 	// control over how long the synchronization task runs.
 	syncer := chain.NewRPCSyncer(wallet, chainClient)
 	go syncer.Run(context.Background(), false)
-	wallet.SetNetworkBackend(chain.BackendFromRPCClient(chainClient.Client))
+
+	n := chain.BackendFromRPCClient(chainClient.Client)
+	wallet.SetNetworkBackend(n)
```

Comment:
why this change?

Reply:
don't recall making this change, reverting